src/Form/Formbot.js

```diff
@@ -125,8 +134,11 @@ export default class Formbot extends React.Component {
         return;
       }
 
-      const fromSchema = !!this.props.validationSchema;
-      const validation = (fromSchema ? this.props.validationSchema : this.props.validations)[field];
+      const { validationSchema, validations } = this.props;
+
+      const fromSchema = !!validationSchema;
+      const schema = typeof validationSchema === 'function' ? validationSchema(this.getContext()) : validationSchema;
```

Comment:
This looks like it may be a slight performance drain to regenerate schema for every field validation. Can we look at providing the Formbot context as the context field in the yup options object we can pass to the validate functions? https://github.com/jquense/yup#mixedvalidatevalue-any-options-object-promiseany-validationerror